components/webui/imports/ui/IngestView/IngestView.scss

```diff
@@ -56,6 +56,8 @@
 
     &-detail {
       display: block;
+      font-family: monospace;
```

Comment:
Could we pick a specific monospaced font? Perhaps Inconsolata or whatever's popular nowadays?

Reply:
For non-source-code text, [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono) might seem less out of space with other fonts.
We can also specify some [Courier New / Courier](https://web.mit.edu/jmorzins/www/fonts.html#monospace) as fallbacks.

Reply:
> We can also specify some [Courier New / Courier](https://web.mit.edu/jmorzins/www/fonts.html#monospace) as fallbacks.

The latest changes use the generic family name `monospace` for fallbacks.